### ISSUE TITLE
210107 : Optimal Training Param

### ISSUE THREAD

**Issue by luwis93choi** [luwis93choi/CNN_VO#1]
Learning Rate : 0.00001 --- Need to verify through learning scheduler

Batch size : 1 --- Need to verify the effect of increased batch size

Epoch : 200
